src/common/sorted-table/index.js

```diff
@@ -354,6 +427,13 @@ export default class SortedTable extends Component {
                   key={id}
                   onClick={rowAction && (() => rowAction(item, userData))}
                 >
+                  {actions != null && <td>
+                    <input
+                      type='checkbox'
+                      onChange={this.linkState(`selectedItems.${id}`)}
+                      checked={Boolean(state.selectedItems[id])}
+                    />
```

Comment:
If `selectedItems` is a `Set`, you won't be able to use `BaseComponent#linkState()`, but you can still mutualize the event handler:

```js
  _selectItem = event => {
    const { checked, name } = event.target
    this.setState({ selectedItems: this.state.selectedItems.add(name) })
  }

  render () {
    return // ...
      <input
        checked={state.selectedItems.has(id)}
        name={id}
        type='checkbox'
      />
   }
```